### ISSUE TITLE
[Enhancement] add selectable option to kirby card

### ISSUE THREAD

**Issue by RasmusKjeldgaard** [kirbydesign/designsystem#3910]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the enhancement
Currently we have a `CardAsButtonDirective` that adds a button role and enter/space keyboard functionality to any card with a click listener added. 

This is somewhat hidden functionality, and will result in nested interactives when a click listener is added alongside other links, buttons etc. inside the card.

At the same time, the card is not easily [labelable](https://developer.mozilla.org/en-US/docs/Web/HTML/Guides/Content_categories#labelable), compared to if a native control was used.

## Describe the solution you'd like
Make an explicit `selectable` input on `kirby-card`, that renders a native button inside the card (similar to kirby-item/ion-item functionality).

If implemented as a [breakout button](https://css-tricks.com/a-complete-guide-to-links-and-buttons/#aa-breakout-buttons) we might mitigate the nested interactive issue all together, or otherwise we can query for nested interactives in card and only render a native button if none are present.

## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Refinement

- [ ] Request that the issue is [UX refined](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#ux-refinement)
- [ ] Request that the issue is [tech refined](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](../CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".

**Comment by jonas-jyskebank-dk** [comment]
Example of issue with labelling in the current implementation of a kirby-card that acts as a button:

```html
<label for="test-button">
   clickadyclick
</label>
<button id="test-button" (click)="onClick()">
   Click me
</button>

<br />

<label for="test-click-card">
   click card
</label>
<kirby-card id="test-click-card" (click)="onClick()">
   Im a click Card!
</kirby-card>
```

With a native button as seen above, you can navigate to the label with e.g. voice-over and it will notify the user that the label is interactable. If the user then uses VO-Space bar it will focus and activate the button, since the label is associated with that control.

In the second case, with a kirby-card that acts as a button with the click handle, this does not happen. Instead the VO-Spacebar on the associated label does not activate the button and it for some reason sets focus in the root of the web view.

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
